api/types/event-emitter.d.ts

```diff
@@ -93,7 +93,7 @@ declare module '@nestjs/event-emitter' {
       object,
       {
         block: BlockFull;
-        passation: Subscriber;
+        passation: Subscriber | null;
```

Comment:
If subscriber is null, we shouldn't emit the event : 
```suggestion
        passation: Subscriber;
```

Reply:
I think the null addition was for typing purposes